typespec-tests/tsp/lro.tsp

```diff
@@ -0,0 +1,117 @@
+import "@azure-tools/typespec-azure-core";
+
+using TypeSpec.Rest;
+using TypeSpec.Http;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.Core.Traits;
+
+@service({
+  title: "LongRunning",
+  version: "1.0.0",
+})
+@useDependency(Azure.Core.Versions.v1_0_Preview_2)
+@route("/long-running")
+namespace LongRunning {
+  // case of LRO having pollingOperation in non-service namespace
+  @pollingOperation(NonService.poll)
+  @post
+  @route("/post")
+  op longRunning(): AcceptedResponse;
```

Comment:
The case of expected way to write typespec when service do not want the op in `@pollingOperation` get generated.

Reply:
There could be a disagreement on whether `PollResponse` model should be in namespace here.

Reply:
I'm also thinking about the namespace of `PollResponse`, as it will be used in the `Client`, so may be better putting it in the @service namespace.

Reply:
OK. Moved PollResponse to service namespace.

Reply:
Both should be same in current processing. However, when switched to TCGC `getAllModels`, this could have a difference.

Also the necessity of the model could be different across language. E.g. Java in `SyncPoller<PollResult, Void>` would need the model, while .NET in `Operation` probably not.